recipes/coremltools/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "coremltools" %}
+{% set version = "4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # The sdist on PyPI lacks at least
+  #   - LICENSE.txt
+  #   - CMakeLists.txt
+  # so we use GitHub releases.
+  url: https://github.com/apple/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: "b254b3b1bdb0d36ea83b34993d5de6d5ca3ab89a93e759d49e7a62307cad694c"
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - numpy
+  host:
+    - python
+    - numpy >=1.14.5
+    - pip
+  run:
+    - python
+    - numpy >=1.14.5
+    - protobuf >=3.1.0
+    - six >=1.1.0
+    - attrs
+    - sympy
+    - scipy
+    - tqdm
+    - packaging
+
+test:
+  imports:
+    - coremltools
+  commands:
+    # TODO: change to running `coremlconverter --help`
+    # when the upstream bug is fixed.
+    - test -f ${PREFIX}/bin/coremlconverter
+
+about:
+  home: https://github.com/apple/coremltools
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: "LICENSE.txt"
+  summary: 'Core ML is an Apple framework to integrate machine learning models into your app. Core ML provides a unified representation for all models.'
+
+  description: |
+    Core ML is an Apple framework to integrate machine learning models into your app.
+
+    Use the coremltools Python package to convert models from third-party training libraries such as TensorFlow and PyTorch to the Core ML format. You can then use Core ML to integrate the models into your app.
+
+    Core ML provides a unified representation for all models. Your app uses Core ML APIs and user data to make predictions, and to fine-tune models, all on the user’s device. Running a model strictly on the user’s device removes any need for a network connection, which helps keep the user’s data private and your app responsive.
+
+    Core ML optimizes on-device performance by leveraging the CPU, GPU, and Neural Engine while minimizing its memory footprint and power consumption.
+  doc_url: https://coremltools.readme.io/docs
+  dev_url: https://github.com/apple/coremltools
+
+extra:
+  recipe-maintainers:
+    - 1duo
```

Comment:
@1duo and @aseem could you comment here to confirm that you're both OK with being maintainers here?

Reply:
Yes, confirmed.

Reply:
Yes.